packages/atomic/src/locales.json

```
@@ -1125,5 +1125,17 @@
   "stars-range": {
     "en": "{{value}} to {{count}} stars",
     "fr": "{{value}} à {{count}} étoiles"
+  },
+  "with-colon": {
+    "en": "{{text}}:",
+    "fr": "{{text}}\u00A0:"
+  },
+  "between-quotations": {
+    "en": "\"{{text}}\"",
+    "fr": "«\u00A0{{text}}\u00A0»"
+  },
+  "between-parentheses": {
+    "en": "({{text}})",
+    "fr": "(\u00A0{{text}}\u00A0)"
```

```suggestion
    "fr": "{{text}}\u00A0: "
  },
  "between-quotations": {
    "en": "\"{{text}}\"",
    "fr": " «\u00A0{{text}}\u00A0» "
  },
  "between-parentheses": {
    "en": "({{text}})",
    "fr": " ({{text}}) "
```

According to the link that @jpmarceau shared with us: http://bdl.oqlf.gouv.qc.ca/bdl/gabarit_bdl.asp?id=2039

---

No need, the extra spaces are already present in the UI. But thanks!